src/template.js

```diff
@@ -36,6 +36,41 @@ function formatValueToList(value) {
   }
 }
 
+function transformObjectData(objectData) {
+  if (getType(objectData) !== 'array') {
+    return objectData;
+  }
+
+  return objectData.map((od) => {
+    if (getType(od) !== 'object') {
+      return od;
+    }
+
+    // The API expects price to be a number so delete it entirely if empty.
+    if (getType(od.price) === 'string' && !od.price) {
```

Comment:
A valid price could be `0`, right? In which case it would be removed. Maybe here we want to check if `od.price !== ''` instead.

Reply:
This was meant to cover, for example, the object:

```js
{
  price: ''
}
```

The intent being that if the user leaves the tag field empty, and therefore the object looks as above, we actually unwind that so the API is happy.

I could change it to be explicit `od.price === ''` if you think that's clearer?

Reply:
Sorry, yes, that's what I meant!